src/service/tags/tags_service.py

```diff
@@ -47,3 +47,10 @@ def get_all_tags(
         except Exception as error:
             self.logger.error(error)
             raise AppError("Can't get tags")
+
+    def get_tags_by_company(self, company_id: str) -> list:
+        try:
+            return self.repository.get_tags_by_company(company_id)
```

Comment:
You can move the validation of the company_id here and raise an exception instead of returning the empty value.